typescript/cli/src/deploy/core.ts

```diff
@@ -484,6 +484,10 @@ async function writeAgentConfig(
     artifacts,
   ) as ChainMap<HyperlaneDeploymentArtifacts>;
 
+  for (const chain of chains) {
+    mergedAddressesMap[chain].interchainGasPaymaster =
+      ethers.constants.AddressZero;
+  }
```

Comment:
seems like we only want to do this when they didn't provide any IGP config, or something along those lines

Reply:
Agreed, what's the value of always including a zero address?